geometry_msgs/Pose pose
string activity

